tchannel-core/src/main/java/com/uber/tchannel/handlers/InitRequestHandler.java

```diff
@@ -64,6 +76,7 @@ public void messageReceived(ChannelHandlerContext ctx, Message message) throws E
 
             default:
 
+                // TODO: should send back BadRequest
```

Comment:
I'm not so sure. Bad Request is defined as `Request args do not match expectations, request will never be satisfied, do not retry.`  InitRequest doesn't even have `args`.